tests/test_request_models.py

```diff
@@ -45,7 +44,7 @@ class TestRequestModel(unittest.TestCase):
     def test_valid_request_with_text_inputs(self):
         data = {
             "inputs": [{"text": "First text"}, {"text": "Second text"}],
-            "data_type": DataTypes.TEXT,
+            "data_type": DataTypes.TEXT.name,

```

Comment:
these should be the enum string values, so that on spreading (with `**data`), they get converted to the correct enum by pydantic.